programs/compute-budget-bench/benches/compute_budget.rs

```diff
@@ -2,14 +2,21 @@ use {
     criterion::{black_box, criterion_group, criterion_main, Criterion},
     solana_compute_budget::compute_budget_limits::ComputeBudgetLimits,
     solana_runtime_transaction::instructions_processor::process_compute_budget_instructions,
-    solana_sdk::{compute_budget::ComputeBudgetInstruction, instruction::CompiledInstruction},
+    solana_sdk::{
+        compute_budget::ComputeBudgetInstruction, feature_set::FeatureSet,
+        instruction::CompiledInstruction,
+    },
     solana_svm_transaction::instruction::SVMInstruction,
     std::num::NonZero,
 };
 
 const ONE_PAGE: u32 = 32 * 1024;
 const SIXTY_FOUR_MB: u32 = 64 * 1024 * 1024;
 
+fn feature_set() -> FeatureSet {
```

Comment:
The benches should create the `feature_set()` outside of the bench run to avoid creating the feature hashmap each iteration